Failed at the lame@1.2.3 install script 'node-gyp rebuild'.

I've tried "npm rebuild lame" and various other things on different help pages, but I can't make this error go away. More details pasted below. Can you tell me the latest versions of node and npm for which lame works? Thanks!

OS X 10.9.5,
Xcode 6.2,
node 6.0.0,
npm 3.8.9

4 errors generated.
make: *** [Release/obj.target/bindings/src/bindings.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Darwin 13.4.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/tomthecollins/Shizz/repos/fj-admin/node_modules/lame
gyp ERR! node -v v6.0.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 
npm WARN fj-admin@1.0.0 No repository field.
npm ERR! Darwin 13.4.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.0.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE

npm ERR! lame@1.2.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lame@1.2.3 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lame package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lame
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lame
npm ERR! There is likely additional logging output above.

nodejs v6
ubuntu 16.04
```
  AR(target) Release/obj.target/deps/lame/liblamevectorroutines.a
  COPY Release/liblamevectorroutines.a
  CXX(target) Release/obj.target/bindings/src/bindings.o
In file included from ../src/bindings.cc:19:0:
../../nan/nan.h:590:20: error: variable or field ‘AddGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback
                    ^
../../nan/nan.h:590:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback
       ^
../../nan/nan.h:591:18: error: expected primary-expression before ‘gc_type_filter’
     , v8::GCType gc_type_filter = v8::kGCTypeAll) {
                  ^
../../nan/nan.h:596:20: error: variable or field ‘RemoveGCEpilogueCallback’ declared void
       v8::Isolate::GCEpilogueCallback callback) {
                    ^
../../nan/nan.h:596:7: error: ‘GCEpilogueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCEpilogueCallback callback) {
       ^
../../nan/nan.h:601:20: error: variable or field ‘AddGCPrologueCallback’ declared void
       v8::Isolate::GCPrologueCallback callback
                    ^
../../nan/nan.h:601:7: error: ‘GCPrologueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCPrologueCallback callback
       ^
../../nan/nan.h:602:18: error: expected primary-expression before ‘gc_type_filter’
     , v8::GCType gc_type_filter = v8::kGCTypeAll) {
                  ^
../../nan/nan.h:607:20: error: variable or field ‘RemoveGCPrologueCallback’ declared void
       v8::Isolate::GCPrologueCallback callback) {
                    ^
../../nan/nan.h:607:7: error: ‘GCPrologueCallback’ is not a member of ‘v8::Isolate’
       v8::Isolate::GCPrologueCallback callback) {
       ^
bindings.target.mk:105: recipe for target 'Release/obj.target/bindings/src/bindings.o' failed
make: *** [Release/obj.target/bindings/src/bindings.o] Error 1
make: Leaving directory '/home/sanya/Work/alexa/AlexaPi/node_modules/lame/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
gyp ERR! System Linux 4.4.0-22-generic
gyp ERR! command "/usr/bin/nodejs" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/sanya/Work/alexa/AlexaPi/node_modules/lame
gyp ERR! node -v v6.2.1
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok 
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.12
npm ERR! Linux 4.4.0-22-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "--save" "https://github.com/kauffecup/node-microphone.git"
npm ERR! node v6.2.1
npm ERR! npm  v3.9.3
npm ERR! code ELIFECYCLE

npm ERR! lame@1.2.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lame@1.2.3 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lame package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lame
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lame
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/sanya/Work/alexa/AlexaPi/npm-debug.log
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -18,6 +18,8 @@ node_js:
   - "iojs-v1"
   - "iojs-v2"
   - "iojs-v3"
+  - "4"
+  - "6"
 
 install:
   - PATH="`npm bin`:`npm bin -g`:$PATH"

--- a/package.json
+++ b/package.json
@@ -19,13 +19,13 @@
   },
   "main": "./index.js",
   "dependencies": {
-    "bindings": "~1.2.0",
-    "debug": "2",
-    "readable-stream": "1.0",
-    "nan": "~2.3.5"
+    "bindings": "^1.2.1",
+    "debug": "^2.2.0",
+    "readable-stream": "^1.0.34",
+    "nan": "^2.3.5"
   },
   "devDependencies": {
-    "mocha": "*"
+    "mocha": "^2.4.5"
   },
   "scripts": {
     "test": "mocha --reporter spec"

